### ISSUE TITLE
Be explicit about which items are included in retention policy?

### ISSUE THREAD

**Issue by qubika** [MicrosoftDocs/microsoft-365-docs#2227]
[Enter feedback here]
Re: "Excluding specific types of Exchange items from a retention policy", can you be explicit about the items which are included in the retention policy?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 3a1c2e44-357c-8096-5bdf-40e205778ded
* Version Independent ID: 19c7b5d9-1e0c-501b-7b70-df5aab48b36e
* Content: [Learn about retention for Exchange - Microsoft 365 Compliance](https://docs.microsoft.com/en-us/microsoft-365/compliance/retention-policies-exchange?view=o365-worldwide)
* Content Source: [microsoft-365/compliance/retention-policies-exchange.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/compliance/retention-policies-exchange.md)
* Service: **o365-seccomp**
* GitHub Login: @cabailey
* Microsoft Alias: **cabailey**

**Comment by cabailey** [comment]
Looking into this & waiting confirmation from engineering

**Comment by cabailey** [comment]
This section needs to clarity that this PowerShell option is only if the retention action is retain-only (not retain and then delete, or delete-only) and I'm told that by default, retention for Exchange includes all items in a user mailbox with these exceptions: Contacts, calendar and tasks with no end date, documents, and distribution lists. I'll add this clarification to the documentation. 

Is that sufficient or are you looking for the list of items that can be excluded with the PowerShell parameter ExcludedItemClasses?

**Comment by qubika** [comment]
The clarification is definitely useful, but for completeness I think it still needs to be explicit about the excludeditemclasses which can be used.

The reference page which is linked shows all of the classes, but I presume not all values are valid.

**Comment by cabailey** [comment]
@qubika  For the Exchange locations, I've got confirmation that mail messages (includes drafts) with any attachments, tasks and calendar items when they have an end date, and notes are included.  I've added this to the page.  I'll follow up separately for a list of items that can be excluded by using PowerShell - the documentation for the ExcludedItemClasses parameter is not at all clear (https://docs.microsoft.com/en-us/powershell/module/exchange/set-retentioncompliancerule?view=exchange-ps#parameters)

**Comment by cabailey** [comment]
@qubika  As a follow-up, the PowerShell description has now been updated to add a link to the message classes for Exchange but also discovered there is no validation on the values entered because it allows for customization.  That's why it can only talk about "examples".  Everything I've learned about this parameter (retain-only, no set list of values, no validation) leads me to conclude that this is not a standard configuration and for specialized use only, so I'm going to remove this section from the core docs.
